benchmark/_interfaces.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
```

Comment:
I think that I picked that template from `benchmark` folder in `flocker`. But I see that the prevailing majority of the code does not use the year. So, I am fixing this.